libs/core/src/components/sage-tab/sage-tab.scss

```diff
@@ -0,0 +1,177 @@
+sage-tab {
+  display: inline-block;
+  position: relative;
+}
+
+///
+/// Resets base button styles
+///
+@mixin sage-button-style-reset {
+  appearance: none;
+  background-color: transparent;
+  border: 0;
+  box-shadow: none;
+  color: inherit;
+  cursor: pointer;
+  font-family: inherit;
+  padding: 0;
+}
+
+// delete later
```

Comment:
when would be an appropriate time to delete it?

Reply:
I've opened a ticket for this work: https://kajabi.atlassian.net/browse/DSS-345